engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -18,3 +18,6 @@
 --;
 -- Schema upgrade from 4.14.0.0 to 4.15.0.0
 --;
+
+-- mysql8 nics table fix for newer distributions
+ALTER TABLE nics MODIFY COLUMN update_time timestamp NULL;
```

Comment:
@davidjumani this was proposed only as a workaround, please explore a proper fix (why we see update_time related sql exceptions on freshly deployed env, i.e. zone with ssvm/cpvm launching).